tcms/rpc/tests/test_testexecutionstatus.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=attribute-defined-outside-init, invalid-name, objects-update-used
+
+from xmlrpc.client import ProtocolError
+from tcms.testruns.models import TestExecutionStatus
+from tcms.rpc.tests.utils import APITestCase, APIPermissionsTestCase
+
+
+class TestFilter(APITestCase):
+    """Test TestExecutionStatus.filter"""
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
```

Comment:
no need for redefining a method only to call it's parent. pylint also tells you that:
Useless super delegation in method '_fixture_setup'